from django.core.exceptions import ValidationError
from django.utils.translation import ugettext_lazy as _

def is_valid_dbf(dbf_file):
    # This is a placeholder for now. We will bring the validation functionality
    # from `load_sinan.py` to this function.
    return True
